src/Mvc/test/WebSites/HtmlGenerationWebSite/Views/HtmlGeneration_Home/Image.cshtml

```diff
@@ -11,27 +11,27 @@
 
     <h2>Image Tag Helper Test</h2>
     <!-- Plain image tag -->
-    <img src="~/images/red.png" alt="Red block" title="&lt;the title>">
+    <img src="~/images/red.png" alt="Red block" title="&lt;the title>" id="1">
```

Comment:
nit: is it worth making these somewhat more descriptive?

Reply:
i.e. instead of id 1-7

Reply:
It's kinda only here because we wrote 7 different test cases in to a single one. I really don't like this test, but I don't have the patience to clean it up either.